.pipelines/v2/templates/job-test-project.yml

```diff
@@ -61,13 +61,11 @@ jobs:
         reg add "HKLM\Software\Policies\Microsoft\Edge\WebView2\ReleaseChannels" /v PowerToys.exe /t REG_SZ /d "3"
       displayName: "Enable WebView2 Canary Channel"
 
-  - download: current
-    displayName: Download artifacts
-    artifact: $(TestArtifactsName)
-    patterns: |-
-      **
-      !**\*.pdb
-      !**\*.lib
+  - template: steps-download-artifacts-with-azure-cli.yml
+    parameters:
+      artifactName: $(TestArtifactsName)
+      downloadPath: $(Pipeline.Workspace)/$(TestArtifactsName)
+      downloadRunId: $(Build.BuildId)
```

Comment:
If you make these parameters default values inside `steps-download-artifacts...`, you can leave them out here _and_ it will be easier to use it as a "drop in" replacement in other projects!

Reply:
updated